plugins/seccomp/src/PolicyView.tsx

```diff
@@ -0,0 +1,171 @@
+import { getSeccompResult, SeccompProfilesApi } from './api';
+import { parseSeccompAnnotations } from './helper';
+import SeccompPolicyInitialView from './InitialView';
+import SeccompPoliciesTableView from './PoliciesTable';
+import SeccompPolicyPolicyGatheringView from './PolicyGatheringView';
+import SeccompPolicyResultView from './PolicyResultView';
+
+const pluginLib = window.pluginLib;
+const { Typography } = pluginLib.MuiCore;
+const { SectionBox, Loader } = pluginLib.CommonComponents;
+const React = pluginLib.React;
+const { useParams } = pluginLib.ReactRouter;
+const K8s = pluginLib.K8s.ResourceClasses;
+const { useSnackbar } = pluginLib.Notistack;
+
+enum SeccompStatus {
+  SECCOMP_NOT_STARTED = -1,
+  SECCOMP_STOPPED,
+  SECCOMP_STARTED,
+}
+
+export default function SeccompPolicyView() {
+  const { namespace } = useParams();
+  const [seccompStatus, setSeccompStatus] = React.useState(SeccompStatus.SECCOMP_NOT_STARTED);
+  const [seccompDefaultTimestamp, setSeccompDefaultTimestamp] = React.useState(null);
+  const [seccompFinalTimestamp, setSeccompFinalTimestamp] = React.useState(null);
+  const [seccompStartTimestamp, setSeccompStartTimestamp] = React.useState(null);
+  const [seccompPolicies, setSeccompPolicies] = React.useState(null);
+  const [appliedPolicies, setAppliedPolicies] = React.useState(null);
+  const [danglingPolicies, setDanglingPolicies] = React.useState(null);
+  const [pods] = K8s.Pod.useList();
+  const { enqueueSnackbar } = useSnackbar();
+  const [currentTracingNamespace, setCurrentTracingNamespace] = React.useState(null);
+
+  function listPolicies() {
+    SeccompProfilesApi.list(
+      namespace,
+      (data: any) => {
+        const massagedData = data?.map((item: any) => ({
+          ...item,
+          pod: pods.find(
+            (pod: any) => pod.metadata.name === parseSeccompAnnotations(item, 'pod', namespace)
```

Comment:
There are lots of `any` types used in this PR. Those should be kept to a minimum, so hopefully this can be improved when the PR is in a more mature state.